src/components/dialogs/parameters/sensi/sensitivity-analysis-parameters.js

```diff
@@ -279,10 +298,10 @@ export const SensitivityAnalysisParameters = ({
                     response.text().then((value) => {
                         setValue(
                             `${arrayFormName}[${index}].[${COUNT}]`,
-                            value && Math.abs(value)
+                            parseIntData(value, 0)
                         );
                         setAnalysisComputeComplexity(getResultCount());
-                        onFormChanged(false);
+                        setLaunchLoader(false);
                     });
                 })
                 .catch((error) => {
```

Comment:
If you put it in the then only, you'll have an infinite loading if you receive an error. Either you add it in the catch as well, or you place it after the then and catch

Reply:
done